pkg/cli/instances.go

```diff
@@ -91,6 +111,14 @@ func InstancesSsh(rack sdk.Interface, c *stdcli.Context) error {
 		opts.Command = options.String(command)
 	}
 
+	if key := c.String("key"); key != "" {
```

Comment:
Maybe check if it's a v3 rack before looking for the key?

Reply:
added the check